src/App.tsx

```diff
@@ -1,24 +1,134 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+
+import { Select } from './types/Select';
+import { Todo } from './types/Todo';
+import { TodoData } from './types/TodoData';
+import { client, todoUrlEnd } from './utils/fetchClient';
+import {
+  createTodo,
+  deleteTodo,
+  changeTodo,
+} from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 10364;
```

Comment:
Probably you need to put it in a separate file like Pasha in his morning lesson

Reply:
Hi! You are right, it isn`t finished, but I made PR bc I had to share it with Lilia.
Thanks for comments, they are very usefull!